lib/manageiq/environment.rb

```diff
@@ -54,9 +53,13 @@ def self.while_updating_ui
       end
 
       ui_thread.abort_on_exception = true
+      ui_thread
+    end
 
+    def self.while_updating_ui
+      # Run update:ui in a thread and continue to do the non-js stuff
+      update_ui_thread
```

Comment:
`ui_thread = update_ui_thread`

otherwise line 63 will :boom:

Reply:
:+1: thanks, this isn't used anymore but didn't want to just delete it yet

Reply:
it is, from `bin/update`

Reply:
Not anymore https://github.com/ManageIQ/manageiq/pull/20018/files#diff-02b49c97a1c0f2b019fdbba32b895c81R24

Reply:
That's my point, `bin/setup` != `bin/update`

Reply:
Ohh :+1: I missed that you said bin/update not bin/setup